src/boot/efi/console.c

```diff
@@ -149,9 +149,69 @@ static EFI_STATUS change_mode(UINTN mode) {
         return err;
 }
 
+static UINT64 text_area_from_font_size()
+{
+        const UINT64 SYSTEM_FONT_WIDTH = 8;
+        const UINT64 SYSTEM_FONT_HEIGHT = 19;
```

Comment:
hmm, this feels like a C++ism: in C that's usually not how people do constants, but instead they use `#defines`

Reply:
Define inside the function or top level ?

Reply:
the systemd codebase defines it top level.